src/NuGet.Core/NuGet.CommandLine.XPlat/Utility/ProjectPackagesPrintUtility.cs

```diff
@@ -132,9 +134,7 @@ internal static class ProjectPackagesPrintUtility
                        p => PrintVersion(
                                 p.ResolvedPackageMetadata.Identity.Version,
                                 p.ResolvedPackageMetadata.DeprecationMetadata != null),
-                       p => p.LatestPackageMetadata?.Identity?.Version == null
-                            ? Strings.ListPkg_NotFoundAtSources
-                            : PrintVersion(
+                       p => PrintVersion(
```

Comment:
There are situations in which printing `Not found at sources` is valid. 
It's possible that the sources have been changed since the last restore.

Reply:
ah, so I should test specifically for package refs, then. I'll figure out how to do that :)

Reply:
Fixed by adding a Type field to the package objects and checking for that.